DirectProgramming/DPC++FPGA/ReferenceDesigns/anr/README.md

```diff
@@ -191,14 +191,11 @@ The following source files can be found in the `src/` sub-directory.
 |`data_bundle.hpp`                | A generic library for bundling data to move between kernels; essentially an array.
 |`dma_kernels.hpp`                | Contains kernels that move data between the host and device, as well as reading/writing data between the FPGA and device memory.
 |`intensity_sigma_lut.hpp`        | A RAM LUT for the intensity sigma values.
-|`mp_math.hpp`                    | Metaprogramming math helper functions.
```

Comment:
I don't think common include files should be removed entirely from the README - we should just point out that they live in a separate directory.  I believe that's what we have done for other samples that use the common files.

Reply:
Addressed in most recent commit.
I put a little sentence after that says "for this, this, and this, file see INCLUDE DIRECTORY".